less/index.less

```diff
@@ -53,7 +53,188 @@ table {
     box-sizing: border-box;
 }
 
+//less variables
+@primary-color: lightgrey;
+
 html, body {
-    height: 100%;
-}
+	height: 100%;
+	width: 95%;
+	margin: 0 auto;
+	// font-size: 62.5%;
+	
+	.nav {
+		display: flex;
+		justify-content: flex-end;
+		padding: 2rem;
+
+		.nav-items {
+			display: flex;
+			justify-content: space-around;
+
+			a {
+				text-decoration: none;
+				font-size: 1.5rem;
+				font-weight: bold;
+				color: black;
+			}
+		}
+	
+	}
+	.header {
+		background-color: @primary-color;
+
+		div {
+			margin: 2rem;
+
+			img {
+				padding: 3rem 0 0.3rem;
+			}
+			h1 {
+				font-size: 4rem;
+				padding-bottom: 1rem;
+			}
+			h3 {
+				font-size: 2rem;
+				padding-bottom: 3rem;
+			}
+			p {
+				padding: 2rem 0;
+			}
+			button {
+				width: 10rem;
+				height: 3rem;
+				border-radius: 0.3rem;
+				background-color: grey;
+				color: white;
+				font-weight: bold;
+				font-size: 1rem;
+				margin-bottom: 2rem;
+				outline: none;
+			}
+		}
+
+	}
+	.top-section {
+		display: flex;
+
+		.left-box {
+			width: 50%;
+			padding-right: 2%;
+
+			h2 {
+				font-size: 2.5rem;
+				padding-bottom: 0.5rem;
+			}
+			p {
+				font-size: 1.3rem;
+			}
+		}
+		.right-box {
+			width: 50%;
+
+			h3 {
+				font-size: 2rem;
+				padding-bottom: 0.5rem;
+			}
+			ul {
+				padding: 2% 0;
+				border: 1px solid @primary-color;
+				li {
+					padding: 1rem;
+					border-bottom: 1px solid @primary-color;
+					font-size: 1.3rem;
 
+					&:last-child {
+						border-bottom: none;
+					}
+				}
+			}
+		}
+	}
+	.bottom-section {
+		padding: 2rem 0;
+
+		h3 {
+			font-size: 2rem;
+			padding-bottom: 0.5rem;
+		}
+		table {
+			width: 100%;
+			font-size: 1.3rem;
+
+			.tb-head {
+				background-color: white;
+				
+				td {
+					padding: 0.5rem;
+					font-weight: bold;
+				}
+			}
+
+			tbody {
+				td {
+					padding: 1rem;
+				}
+				tr:nth-child(odd) {
+					background-color: @primary-color;
+				}
+				
+			}
+
+		}
+	}
+	.footer {
+		display: flex;
+		margin-top: 2rem;
+		width: 100%;
+		height:6rem;
+		background-color: grey;
+		color: white;
+		justify-content: center;
+		align-items: center;
+
+		p {
+			font-size: 1rem;
+			font-weight: bold;
+		}
+	}
+	@media (max-width: 500px) {
```

Comment:
You could make max-width:500px a variable like @phone or @mobile so you can adjust your breakpoint without manually updating each media query :)